components/SessionPreview/index.js

```diff
@@ -29,13 +36,17 @@ export default function SessionPreview({ template }) {
 
   return (
     <PreviewContainer>
-      <section>{template.name}</section>
-      <DayDisplay>{`Day ${dayCount + 1}`}</DayDisplay>
-      <ExercisesPreviewContainer>
-        {template.routine[0].exercises.map((exercise) => (
-          <li key={exercise.id}>{exercise.exercise}</li>
-        ))}
-      </ExercisesPreviewContainer>
+      <StyledCard>
+        <section>{template.name}</section>
```

Comment:
```suggestion
        <span>{template.name}</span>
```

A section for just a word is not necessary :)
You can use a span or a p tag.